packages/design-tokens/src/scss/generated/_typography.scss

```diff
@@ -1,45 +1,32 @@
 // Generated Typography from Supernova. Do not edit manually.
-$heading-xlarge-text: (
+$body-large-button: (
     mobile: (
         font-family: "'Inter', sans-serif",
-        font-size: 3rem,
-        font-style: normal,
-        font-weight: 700,
-        line-height: 1.2,
-    ),
-    tablet: (
-        font-family: "'Inter', sans-serif",
-        font-size: 4rem,
-        font-style: normal,
-        font-weight: 700,
-        line-height: 1.2,
-    ),
-    desktop: (
-        font-family: "'Inter', sans-serif",
-        font-size: 4rem,
+        font-size: 1.125rem,
         font-style: normal,
-        font-weight: 700,
-        line-height: 1.2,
+        font-weight: 600,
+        line-height: 1.5,
     ),
 ) !default;
 
-$body-medium-text-regular: (
+$body-large-link-no-underline: (
```

Comment:
As @literat pointed out, these links are still underlined. I think the most efficient way is to globally reset link underline in `foundation/_reset.scss`.

![obrazek](https://user-images.githubusercontent.com/5614085/159452829-ad7b8f94-eb2e-43f6-8856-8ed89dfbf750.png)